.gitignore

```diff
@@ -1,4 +1,4 @@
-# extra files created by cmake
+z# extra files created by cmake
```

Comment:
```suggestion
# extra files created by cmake
```

Reply:
Resolved in [68c65bf](https://github.com/NNPDF/nnpdf/pull/1816/commits/68c65bf8507faaf10ab8c33687125d8ef201fd67).